src/components/cdPipeline/cdPipeline.types.ts

```diff
@@ -26,7 +26,8 @@ export interface CDPipelineProps
     appName: string;
     downstreamNodeSize: number;
     getWorkflows: () => void;
-    close: (showSuccessCD?: boolean, environmentId?: number, environmentName?: string, successTitle?: string) => void;
+    close: (showSuccessCD?: boolean, environmentId?: number, environmentName?: string, successTitle?: string, showWebhookTippy?: boolean) => void;
+    deleteWorkflow?: (appId?: string, workflowId?: number) => any
```

Comment:
Can we specify the proper type instead of any, if possible?

Reply:
done